asdf/tests/test_entry_points.py

```diff
@@ -23,7 +23,7 @@ def _monkeypatch_entry_points(monkeypatch, mock_entry_points):
     def _entry_points(*, group):
         for candidate_group, name, func_name in mock_entry_points:
             if candidate_group == group:
-                point = metadata.EntryPoint(name=name, group="asdf.tests.test_entry_points", value=func_name)
+                point = metadata.EntryPoint(name=name, group=group, value=func_name)
```

Comment:
This looks like it actually fixes a "bug" in the tests?

Reply:
The group attribute of the returned EntryPoints was unused prior to this PR. I don't know what the intention of the previous group constant was but it had no consequence.